mempool/policy_test.go

```diff
@@ -101,8 +102,9 @@ func TestCalcMinRequiredTxRelayFee(t *testing.T) {
 func TestCheckPkScriptStandard(t *testing.T) {
 	var pubKeys [][]byte
 	for i := 0; i < 4; i++ {
-		pk := chainec.Secp256k1.NewPrivateKey(big.NewInt(int64(chainec.ECTypeSecp256k1)))
-		pubKeys = append(pubKeys, chainec.Secp256k1.NewPublicKey(pk.Public()).SerializeCompressed())
+		pk := secp256k1.NewPrivateKey(big.NewInt(int64(chainec.ECTypeSecp256k1)))
```

Comment:
I don't see any reason to retain the reference to `chainec` here.  It's just generating a new private key from a fixed value for testing purposes.  You can replace it with `big.NewInt(0)`.